elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManager.java

```diff
@@ -499,6 +523,16 @@ public String getActualIndexName() {
 		return actualIndexName;
 	}
 
+	@Override
+	public Analyzer createAnalyzer(String name) {
+		return new ElasticsearchAnalyzer( name );
```

Comment:
N.B. AnalyzerDef is not an annotation which makes much sense to have on a field.

Remember it purely *defines* the analyzer, it doesn't apply it.

I see the source code lists

    @Target({ ElementType.PACKAGE, ElementType.TYPE, ElementType.FIELD, ElementType.METHOD })

but I think that's a mistake.
Think of it as something similar to `@NamedQuery`.